src/lib/utils/index.ts

```diff
@@ -74,6 +74,15 @@ export const formatDollarAmount = (
   }).format(amount)
 }
 
+export function formatAmountFromWei(
+  wei: bigint,
+  units: number = 18,
```

Comment:
Maybe make units mandatory to avoid people forgetting to set decimals? We use a lot of USDC/DAI/etc. so most likely always need to set it.

Reply:
Yeah making `decimals` mandatory would be my preference - makes us double check the value we use. I'm not as opinionated on `digits` though...feels 'safer' to have a default there